codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -469,8 +469,99 @@ class ServerServiceGeneratorV2(
     }
 
     fun render(writer: RustWriter) {
+        val handlers: Writable = operations
+            .map { operation ->
+                DocHandlerGenerator(operation, "///", builderFieldNames[operation]!!, codegenContext).docSignature()
+            }
+            .reduce { acc, wt ->
+                writable {
+                    rustTemplate("#{acc:W} \n#{wt:W}", "acc" to acc, "wt" to wt)
+                }
+            }
+
         writer.rustTemplate(
             """
+            /// This module contains the implementation of $serviceName.
+            /// [`$serviceName`] is used to set your business logic to implement your [operations],
```

Comment:
> used to set your business logic to implement your

I think this description should be used in reference to the builder. The `$serviceName` is used as a `Service` we provide to lambda or `hyper` post-`into_make_service` - it does contain business logic but it's not the API we use to set the business logic.